src/commands/sbt_cached.yml

```
@@ -149,7 +149,7 @@ steps:
               find $HOME/.sbt -name "*.lock" | xargs rm | true
               find $HOME/.ivy2 -name "ivydata-*.properties" | xargs rm | true
               # delete all jars inside target directories
-              find . -name *.jar -type f -exec rm {} \;
+              find . -type f -name *.jar ! -name *-assembly.jar -exec rm {} \;
```

Should this not contain quotes? So it would be as follows:
```suggestion
              find . -type f -name *.jar ! -name "*-assembly.jar" -exec rm {} \;
```